rules/gluster-volume.libsonnet

```diff
@@ -0,0 +1,17 @@
+{
+  prometheusRules+:: {
+    groups+: [
+      {
+        name: 'gluster-volume.rules',
+        rules: [
+          {
+            expr: |||
+               sum(max(gluster_subvol_capacity_used_bytes{%(glusterExporterSelector)s}) BY (volume, subvolume)) BY (volume)
```

Comment:
I know this PR is merged and I missed to review this PR. Can you please explain the logic behind doing a 'max'  here. @umangachapagain

Reply:
@cloudbehl @shtripat @umangachapagain Shouldn't it be just : sum(gluster_subvol_capacity_used_bytes)by(volume)

Reply:
http://aravindavk.in/blog/monitoring-glusterfs-volume-utilization/
This should clear your doubts.

Reply:
Shouldn't this be enough : ` max(gluster_subvol_capacity_used_bytes{job="glusterd2-client"})by(volume) `

Reply:
Consider a Volume made of 2 - replica 3 subvolumes.
To get `used bytes` of this `volume` we need to `sum used bytes` of each `subvolume` in a volume. Hence, `sum by volume`
To get `used bytes` of each `subvolume`, we need to get `max` of `used bytes` of `replica of each subvolumes` for each `volume`. Hence, `sum by subvolume and volume`

Reply:
I tested. Looks Good. Thanks for explaining